Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,11 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.app {
+  height: 500px;
+  width: 45%;
+  margin: 1em auto;
+  border: 0.5px solid #000;
```

Comment:
`border: 0.5px` visually makes no difference in the line but actually reduces the padding inside the content. The thinnest line you can have is at `1px`